src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -892,12 +893,19 @@ public virtual void remove_rollback_directory_if_exists(string packageName)
 
             var originalConfig = config;
 
+            var notFoundedMessage = new StringBuilder();
+
             foreach (var packageName in packageNames)
             {
                 // reset config each time through
                 config = originalConfig.deep_copy();
 
                 var installedPackage = packageManager.LocalRepository.FindPackage(packageName);
+                if (installedPackage == null)
+                {
+                    notFoundedMessage.AppendLine("Package {0} was not founded.".format_with(packageName));

```

Comment:
Typo at  "not founded", should be "not found". However, this doesn't come back like we expect. If someone is calling with limit output, we need to adjust to only provide the right output. We also have a particular exit code we set for enhanced exit codes when things are not found.

Reply:
Thanks! One question, if the user specified `ignore-unfound`, should we still use the exit code you mentioned? Thanks!